app/src/main/java/com/meewii/rateconverter/business/Exceptions.kt

```diff
@@ -0,0 +1,11 @@
+package com.meewii.rateconverter.business
+
+/**
+ * Error returned when the API returns a response that contains invalid data
+ */
+class InvalidResponseException(override val message: String) : Exception(message)
```

Comment:
Having custom exceptions are a neat idea. I would suggest to use a sealed class without inheriting Exception though. Two reasons for that:
* When dealing with Kotlin Exceptions, I would prefer Throwables and this may sometimes conflict with Java Exceptions compatibility.
* Since this is a custom exception that belongs to the this project, I would avoid using language level exceptions, since it may produce unwanted results. For me custom exceptions are a way to deal with view states and I really would not like to deal with language level exceptions, that may crash the app if not handled properly :D